packages/front-end/pages/datasources/[did].tsx

```diff
@@ -260,123 +207,36 @@ mixpanel.init('YOUR PROJECT TOKEN', {
               </p>
 
               <div className="card py-3 px-3 mb-4">
-                {/* TODO: design changes for Identity Joins nested */}
-                {/* region Identifier Types */}
                 <DataSourceInlineEditIdentifierTypes
                   onSave={updateDataSource}
-                  onCancel={cancelUpdateDataSource}
+                  onCancel={() => undefined}
                   dataSource={d}
                 />
-                {/* endregion Identifier Types */}
 
                 <div className="mt-4">
-                  {/* region Identity Joins */}
                   <DataSourceInlineEditIdentityJoins
                     dataSource={d}
                     onSave={updateDataSource}
-                    onCancel={cancelUpdateDataSource}
+                    onCancel={() => undefined}
                   />
                 </div>
-                {/* endregion Identity Joins */}
               </div>
 
-              <div className="mb-4">
-                <h3>Experiment Assignment Queries</h3>
-                <p>
-                  Returns a record of which experiment variation was assigned to
-                  each user.
-                </p>
```

Comment:
The whole Experiment Assignment queries section was moved into the component `<ExperimentAssignmentQueries />` below